packages/http/libraries/core/src/http/adapter/InversifyHttpAdapter.ts

```diff
@@ -205,67 +226,91 @@ export abstract class InversifyHttpAdapter<
     response: TResponse,
     next: TNextFunction,
   ): Promise<unknown[]> {
-    return Promise.all(
+    const params: unknown[] = new Array(
+      controllerMethodParameterMetadataList.length,
+    );
+
+    await Promise.all(
```

Comment:
I think a for loop approach is more explicity for this type of operation. What do You think?

Reply:
Yeah, but on the other hand Promise.all adds less overhead